src/applications/disability-benefits/4142/config/form.js

```diff
@@ -0,0 +1,252 @@
+
+const { fullName, ssn, date, dateRange, usaPhone } = commonDefinitions;
+
+// Define all the fields in the form to aid reuse
+const formFields = {
+  fullName: 'fullName',
+  ssn: 'ssn',
+  toursOfDuty: 'toursOfDuty',
+  privateMedicalProvider: 'privateMedicalProvider',
+  viewNoDirectDeposit: 'view:noDirectDeposit',
+  viewStopWarning: 'view:stopWarning',
+  bankAccount: 'bankAccount',
+  accountType: 'accountType',
+  accountNumber: 'accountNumber',
+  routingNumber: 'routingNumber',
+  address: 'address',
+  email: 'email',
+  altEmail: 'altEmail',
+  phoneNumber: 'phoneNumber',
+};
+
+// Define all the form pages to help ensure uniqueness across all form chapters
+const formPages = {
+  applicantInfo: 'applicantInfo',
+  uploadInfo: 'uploadInfo',
+  serviceHistory: 'serviceHistory',
+  treatmentHistory: 'treatmentHistory',
+  contactInfo: 'contactInfo',
+  directDeposit: 'directDeposit',
+};
+
+const formConfig = {
+  urlPrefix: '/',
+  submitUrl: '/v0/private_medical_record_auth/submit',
+  trackingPrefix: 'complex-form-',
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  formId: '21-4142',
+  version: 0,
+  prefillEnabled: true,
+  //  prefillTransformer, //TODO: DO WE NEED THIS?
+  savedFormMessages: {
+    notFound: 'Please start over to apply for benefits.',
+    noAuth: 'Please sign in again to continue your application for benefits.',
+  },
+  //  transformForSumbit: transform, //TODO
+  title: '4142 Private Medical Record Release Form',
+  defaultDefinitions: {
+    fullName,
+    ssn,
+    date,
+    dateRange,
+    usaPhone,
+  },
+  chapters: {
+    chapterApplicantInfo: {
+      title: 'Apply for disability increase',
+      pages: {
+        [formPages.uploadInfo]: {
+          path: 'prviate-medical-record',
+          title: 'Supporting Evidence',
+          uiSchema: {
+            'view:uploadRecs': {
+              'ui:description': aboutPrivateMedicalRecs,
+              'ui:title': aboutPrivateMedicalRecs,
+              'ui:widget': 'radio',
+              'ui:options': {
+                labels: {
+                  yes: 'Yes',
+                  no: 'No, please get them from my doctor.',
+                },
+              },
+            },
+            'view:privateRecordsChoiceHelp': {
+              'ui:description': medicalRecDescription,
+            },
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              'view:uploadRecs': {
+                type: 'string',
+                'enum': ['Yes', 'No, please get them from my doctor.'],
+              },
+              'view:privateRecordsChoiceHelp': {
+                type: 'object',
+                properties: {},
+              },
+            },
+          },
+        },
+      },
+    },
+    chapterServiceHistory: {
+      title: 'Supporting Evidence',
+      pages: {
+        [formPages.treatmentHistory]: {
```

Comment:
Seems like this page should only show if the veteran replied 'no' on the previous page?

Reply:
This is part of the the fake screen that is just a placeholder. After integration with 526, this will have real functionality.

Reply:
This is not a functioning screen. When it is integrated into 526 it will not be used.